common/migration/1669676905990-AddLastCheckedNotifications.ts

```diff
@@ -0,0 +1,20 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class AddLastCheckedNotifications1669676905990 implements MigrationInterface {
+    name = 'AddLastCheckedNotifications1669676905990'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "pupil" ADD "lastTimeCheckedNotifications" TIMESTAMP DEFAULT '"2022-11-28T23:08:26.067Z"'`);
```

Comment:
I think you need to alter every table in two steps, first using TIMESTAMP DEFAULT NULL to fill the column with null values, than alter DEFAULT to current timestamp for future mutating queries https://www.postgresqltutorial.com/postgresql-date-functions/postgresql-current_timestamp/

Reply:
This was automatically generated, I wouldn't touch this.